app/views/ror/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<% @suggestions.each do |value, label| %>
+  <li class="list-group-item" role="option" data-autocomplete-value="<%= value %>" data-autocomplete-label="<%= label %>"><%= html_escape(label) %></li>
```

Comment:
Is Siteimprove happy with this `ul`/`li` arrangement?

Reply:
It is what is currently used for FAST autosuggest and I don't think it was flagged.